commands/slash/openstack.go

```diff
@@ -0,0 +1,280 @@
+package slash
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/ritsec/ops-bot-iii/commands/slash/permission"
+	"github.com/ritsec/ops-bot-iii/config"
+	"github.com/ritsec/ops-bot-iii/data"
+	"github.com/ritsec/ops-bot-iii/helpers"
+	"github.com/ritsec/ops-bot-iii/logging"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+var (
+	// Environment variables required for openstack CLI
+	OS_AUTH_URL             string = config.GetString("openstack.ENV.OS_AUTH_URL")
+	OS_PROJECT_ID           string = config.GetString("openstack.ENV.OS_PROJECT_ID")
+	OS_PROJECT_NAME         string = config.GetString("openstack.ENV.OS_PROJECT_NAME")
+	OS_USER_DOMAIN_NAME     string = config.GetString("openstack.ENV.OS_USER_DOMAIN_NAME")
+	OS_PROJECT_DOMAIN_ID    string = config.GetString("openstack.ENV.OS_PROJECT_DOMAIN_ID")
+	OS_USERNAME             string = config.GetString("openstack.ENV.OS_USERNAME")
+	OS_PASSWORD             string = config.GetString("openstack.ENV.OS_PASSWORD")
+	OS_REGION_NAME          string = config.GetString("openstack.ENV.OS_REGION_NAME")
+	OS_INTERFACE            string = config.GetString("openstack.ENV.OS_INTERFACE")
+	OS_IDENTITY_API_VERSION string = config.GetString("openstack.ENV.OS_IDENTITY_API_VERSION")
+
+	// Paths for scripts to automate openstack user management
```

Comment:
Just curious, is there a reason why you went with executing a script versus using an [Openstack GO SDK](https://github.com/gophercloud/gophercloud)? I'm wondering if that might be a better approach when it comes to error handling and dependencies (now you would need to ensure the Openstack CLI is setup on the running machine)

Reply:
I also just noticed that the scripts are not checked into the repo. Thoughts on including those?

Reply:
> Just curious, is there a reason why you went with executing a script versus using an [Openstack GO SDK](https://github.com/gophercloud/gophercloud)?

Not gonna lie, I did not think about a GO package for interacting with Openstack API. I should have checked if there is one out of there. I will look into it and see if I can implement it.

> I also just noticed that the scripts are not checked into the repo. Thoughts on including those?

Sure, no problem.